hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormat.java

```diff
@@ -433,46 +434,57 @@ protected Expression getFilterPredicates(Configuration configuration) {
     // First prune using default datamap on driver side.
     DataMapExprWrapper dataMapExprWrapper = DataMapChooser
         .getDefaultDataMap(getOrCreateCarbonTable(job.getConfiguration()), resolver);
-    List<ExtendedBlocklet> prunedBlocklets =
+    List<ExtendedBlocklet> finalPrunedBlocklets =
```

Comment:
It is better to give a different name, initially it is for main index pruning

Reply:
fixed. Use the origin variable name 'prunedBlocklets' to represent the final output